ui/address/AddressAccountHistory.tsx

```diff
@@ -58,7 +57,6 @@ const AddressAccountHistory = ({ scrollRef }: Props) => {
         isLoading={ pagination.isLoading }
       />
 
-      <Pagination ml={{ base: 'auto', lg: 8 }} { ...pagination }/>
```

Comment:
If pagination is no longer used, consider using the regular `useApiQuery` hook instead of `useQueryWithPages`.